src/Search_Replace_Command.php

```diff
@@ -108,6 +108,9 @@ class Search_Replace_Command extends WP_CLI_Command {
 	 * [--regex-delimiter=<regex-delimiter>]
 	 * : The delimiter to use for the regex. It must be escaped if it appears in the search string. The default value is the result of `chr(1)`.
 	 *
+	 * [--regex-limit=<regex-limit>]
+	 * : The maximum possible replacements for the regex in each unserialized data bit per row. Defaults to `-1` (no limit).
```

Comment:
The serialized case is one case only and secondary, so I think `The maximum possible replacements for the regex per row (or per unserialized data bit per row). Defaults to `-1` (no limit).` is better.